src/infrastructure/database/repositories/steam.apps.repository.spec.js

```diff
@@ -0,0 +1,792 @@
+import { SteamApp } from "../../../models/steam.app.js";
+import { initiateInMemoryDatabase } from "../in.memory.database.client.js";
+import { SteamAppsRepository } from "./steam.apps.repository.js";
+
+describe("SteamAppsRepository", function () {
+  describe(".insertManySteamApps adds provided apps into the database", function () {
```

Comment:
Redundant test, you tested in the client that you can add multiple apps. `insertManySteamApps` is just a wrapper.
I would remove all redundant tests. Only if there is logic a test is needed.